sites/funmooc/src/backend/templates/social-networks/blogpost-badges.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n static %}{% spaceless %}
+{% with full_page_url=SITE.web_url|add:page_url %}
+    {% blocktrans with title=page_title asvar mailto_subject %}Richie news: {{ title }}{% endblocktrans %}
+    {% blocktrans with title=page_title|safe url=full_page_url asvar shared_sentence %}Richie news: {{ title }} {{ url }}{% endblocktrans %}
+    <div class="social-network-badges">
+        <a
+            href="https://www.facebook.com/share.php?u={{ full_page_url }}"
+            class="social-network-badges__item"
+            target="_blank"
+            title="{% trans "Share on Facebook" %}"
```

Comment:
I think you need to provide translation this.

Reply:
You pointed the wrong lines, you may want to point the new Linkedin part instead. However i never added any translation on your system, i prefer that you manage it.